templates/careers/company-culture/diversity.html

```diff
@@ -84,7 +84,7 @@ <h3 class="p-muted-heading">Equal opportunities</h3>
 
   <section class="p-strip--diversity u-no-padding--top u-no-padding--bottom">
     <div class="u-fixed-width">
-      <hr class="u-no-margin--bottom" />
+      <hr class="p-rule" />
```

Comment:
this HR is inside the section, so according to the logic you wrote it should be muted, shouldn't it?

Reply:
This hr is at the top of the section. Those are p-rule, all the ones that are not on top/bottom of the section are muted. Sorry if that was unclear
![Screenshot 2024-09-24 at 12 19 25](https://github.com/user-attachments/assets/1a7c26d0-76a7-424d-bca3-f436b26c2c75)